README.md

```diff
@@ -281,6 +281,28 @@ app.Command("list", "list all configs", func(cmd *cli.Cmd)) {
 
 This could go on to any depth if need be.
 
+mow.cli also supports command aliases. For example:
+
+```go
+app.Command("start, run, r", "start doing things", cli.ActionCommand(func() { start() }))
```

Comment:
To staty consistent with options, could you get rid of the comma and use spaces instead as aliases separator ?